src/ztc/api/views/besluittype.py

```diff
@@ -22,6 +23,38 @@
 
 
 @conditional_retrieve()
+@extend_schema_view(
+    list=extend_schema(
+        summary="Alle BESLUITTYPEn opvragen.",
+        description="Deze lijst kan gefilterd wordt met query-string parameters.",
```

Comment:
Zou je alle summaries en descriptions translatable willen maken? Kleine moeite voor ons maar kan handig zijn voor in de toekomst